### PR TITLE
User implementation in event management service

### PR DESCRIPTION
**Pull request by Em-Solo**
Implemented ticket Price and user id in everything 
added new functionality to get all events of a user

### CODE REVIEW

File: Event-Management-Service/src/Schemas/events_schemas.ts [hunk 1]
```diff
@@ -19,29 +19,46 @@ const payload = {
...
-      required_error: 'eventId is required'
+      required_error: 'The events id is required'
     })
   })
 }
 
 const params2 = {
   params: object({
+    authorization: string({
+      required_error: 'The events id is required'
```
Comment (ImogenHay) [inline Event-Management-Service/src/Schemas/events_schemas.ts:43]: Should this be user id?

File: Event-Management-Service/src/Schemas/events_schemas.ts [hunk 2]
```diff
@@ -19,29 +19,46 @@ const payload = {
...
     numOfTickets: string({
       required_error: 'Number of tickets you are trying to buy is required'
     })
   })
 }
 
+const params3 = {
+  params: object({
+    authorization: string({
+      required_error: 'The events id is required'
```
Comment (ImogenHay) [inline Event-Management-Service/src/Schemas/events_schemas.ts:57]: Should this be user id?

Comment (Em-Solo) [reply Event-Management-Service/src/Schemas/events_schemas.ts:57]: should be the same, as above

File: Event-Management-Service/src/Schemas/events_schemas.ts (hunk 1 above)
Comment (ImogenHay) [reply Event-Management-Service/src/Schemas/events_schemas.ts:43]: Maybe 'Auth token is required'?

**Review by ImogenHay — APPROVED** [review]

**Pull request merged**